docs/learn-governance.md

```diff
@@ -281,30 +286,40 @@ support - will move to a public referendum under a neutral, majority-carries vot
 case that all members of the council vote in favor of a motion, the vote is considered unanimous and
 becomes a referundum with negative adaptive quorum biasing.
 
-A two-thirds majority of the council can cancel a referendum. This may function as a last-resort if
-there is an issue found late in a referendum's proposal such as a bug in the code of the runtime
-that the proposal would institute.
+### Canceling
+
+A proposal can be canceled if the [technical committee](#technical-committee) unanimously agrees to
+do so, or if Root origin (sudo or Council) triggers this functionality. A canceled proposal's
+deposit is burned.
+
+Additionally, a two-thirds majority of the council can cancel a referendum. This may function as a
+last-resort if there is an issue found late in a referendum's proposal such as a bug in the code of
+the runtime that the proposal would institute.
 
 If the cancellation is controversial enough that the council cannot get a two-thirds majority, then
 it will be left to the stakeholders _en masse_ to determine the fate of the proposal.
 
-> For information on council, please head over to our
-> [technical explainer video](https://www.youtube.com/watch?v=837Vv3gdRzI&list=PLOyWqupZ-WGuAuS00rK-pebTMAOxW41W8&index=32&ab_channel=Polkadot).
+### Blacklisting
+
+A proposal can be blacklisted by Root origin (sudo or Council). A blacklisted proposal and its
+related referendum (if any) is immediately [canceled](#canceling). Additionally, a blacklisted
+proposal's hash can never re-appear in the proposal queue.
```

Comment:
In what situation would blacklisting occur?

Reply:
I have no idea. In theory, someone trying repeatedly to push a proposal through, but if it failed once I don't see why someone would, especially since failing burns the deposit.

Reply:
Hmm..I see. I think it would be helpful to add in a sentence on perhaps an example of this blacklisting situation.

Reply:
Let me ask around